packages/components/src/Divider/__snapshots__/Divider.test.tsx.snap

```diff
@@ -3,7 +3,7 @@
 exports[`Divider should render 1`] = `
 <div>
   <div
-    class="divider horizontal"
+    class="divider base horizontal"
```

Comment:
The word "base" was added to the snapshot test because the css was refactored to include a base size.